ArcadiaParties.API/appsettings.json

```diff
@@ -1,4 +1,7 @@
 {
+  "ConnectionStrings": {
+    "DefaultConnection": "Data Source=ArcadiaParties.db"
```

Comment:
Where is a file itself?

Reply:
Cause we will use seeding data, DB file creates at run project.

Reply:
Ok.